SQL-Scripts/create_Lang_Corpus.sql

```diff
@@ -0,0 +1,70 @@
+/* Change Log															*/
+/* Alpha - 3/9/2021 - Garrett Reed - Initial Creation - Create English and French Corpus Tables	*/
+
+/*
+Hi Garrett et al  Even though I don't care about this being developed "at scale", I would like it to be "scalable"! 
+
+Your proposed data model would work for a small corpus it doesn't really work when you get to, say, a few million words. 
+Focus on the fast lookup you need. Given a word+POS, you want to retrieve all sentences containing that word. 
+If I were doing this in Python, that might look like a dictionary that has the words/part-of-speech tags as keys, and something like a list of sentence IDs as values:
+
+{ "sink/NOUN": [4, 8, 23, 141], "good/ADJ": [1,2,7,14,16,143], ...}
+
+In short you'd like to be able to reproduce this fast lookup of sentence IDs with your database.
+*/
+use wordsense;
+
+/* This is a good step to include in the case the existing table is corrupted in some way */

```

Comment:
I like the comments, it really helps since I know we aren't all familiar with SQL